projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-tab-edit-modal/dossier-management-widget-tab-edit-modal.scss

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.edit-widget-tab-modal .form {
+  &__row {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__element {
+    width: 100%;
+    margin-bottom: 32px;
```

Comment:
If you want to distance the form from the footer, this should be done from the parent, the section element. Doing it from the element within the form exposes it to future alignment issues, should the template change